playbook/app/pb_kits/playbook/pb_timeline/_item.tsx

```diff
@@ -10,8 +8,8 @@ import IconCircle from '../pb_icon_circle/_icon_circle'
 
 type ItemProps = {
   className?: string,
-  children?: array<React.Node>,
-  date?: date,
+  children?: React.ReactNode[],
```

Comment:
Everything looks great! Just one suggestion, I would make this `React.ReactNode[] | React.ReactNode` instead of just React.ReactNode[]. It's always better to err on side of caution with children type!